packages/fsweb/polyfills/window.js

```diff
@@ -12,3 +12,5 @@ export const screen = {
   width: 1024,
   height: 1024
 }
+
+export const RegExp = () => {};
```

Comment:
`Contentful` library will crash unless there is `RegExp` function defined in `window` global object.